.github/workflows/ci.yml

```diff
@@ -88,15 +88,29 @@ jobs:
         env:
           DATADOG_API_KEY: ${{ secrets.DATADOG_API_KEY_E2E }}
           DATADOG_APP_KEY: ${{ secrets.DATADOG_APP_KEY_E2E }}
-      - name: Run junit upload test
-        run: yarn datadog-ci junit upload --service=datadog-ci-e2e-tests-junit artifacts/e2e/junit-reports
+      - name: Run junit upload test (measures)
+        run: yarn datadog-ci junit upload --service=datadog-ci-e2e-tests-junit artifacts/e2e/junit-reports --measures key1:20 --report-measures key1:20
```

Comment:
Would it make sense to have different measures on the session and test levels? This way we can test them separately! You can then change `EXTRA_QUERY_FILTER` into `EXTRA_SESSION_QUERY_FILTER` and `EXTRA_TEST_QUERY_FILTER`

Reply:
two queries are made for two test levels, so even if the measures are the same, if one of the test levels (session or test) are not assigning the measures correctly, we'll detect it.

Reply:
I think this would be testing the same functionality but I like your idea of having different measures! Ty :)